src/components/DocumentationTopic/OnThisPageStickyContainer.vue

```diff
@@ -37,6 +45,10 @@ export default {
   max-height: calc(100vh - #{$top});
   overflow: auto;
 
+  &.ide {
```

Comment:
You could also use, so you wont need any js changes. 

```scss
@include inTargetIde() {
    margin-left: $nav-padding;
}
```

Reply:
b976359e962a4639a685960412f9ff735d6b705f